spec/features/user_contributes_new_councillor_spec.rb

```diff
@@ -18,6 +18,56 @@
       end
     end
 
+    scenario "on the contribution page" do
```

Comment:
Does this feel like a view spec rather than an integration test? https://robots.thoughtbot.com/how-we-test-rails-applications#view-specs

It doesn't really go through a user flow, and the scenario description is confusing to me. It implies that there's other places to make this contribution and doesn't refer to what this is actually testing: whether the name of the authority is on the page. What do you think @hisayohorie ?